diff --git a/node_modules/@weknow/gatsby-remark-drupal/gatsby-node.js b/node_modules/@weknow/gatsby-remark-drupal/gatsby-node.js
index 0dc062c..e8a10ad 100644
--- a/node_modules/@weknow/gatsby-remark-drupal/gatsby-node.js
+++ b/node_modules/@weknow/gatsby-remark-drupal/gatsby-node.js
@@ -33,7 +33,7 @@ exports.onCreateNode = ({
         } = actions;
 
         let content = node.body.value;
-        const inlineImageRegExp = /\(\/sites[^)]+\)/gi;
+        const inlineImageRegExp = /\"\/sites[^\"]+\"/gi; 
         const nodeImages = content.match(inlineImageRegExp);
         if (nodeImages) {
             const nodes = getNodes();
